cmd/copy.go

```diff
@@ -0,0 +1,111 @@
+//
+// DISCLAIMER
+//
+// Copyright 2020 ArangoDB GmbH, Cologne, Germany
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+// Author Gergely Brautigam
+//
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+
+	"github.com/arangodb-managed/arangocopy/pkg"
+)
+
+func init() {
+	InitCommand(
+		RootCmd,
+		&cobra.Command{
+			Use:   "copy",
+			Short: "Copy data from source to destination.",
+		},
+		func(c *cobra.Command, f *flag.FlagSet) {
+			cargs := &struct {
+				sourceAddress          string
```

Comment:
Use struct (probably from ../pkg) for source & destination.
This struct must contain address & auth info